src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -948,6 +948,92 @@ It is advised to embed credentials and certificates in the configuation file, in
 
 - *image:* (Optional) Docker image used to deploy ProActive Nodes. When it is not set, its default value `java:8-jre` is used.
 
+- *nodeDeploymentTemplateFile:* (Optional) Kubernetes deployment template file for deploying ProActive node pods. This template file tells Kubernetes platform the desired running environment for the ProActive node. Each ProActive node will use this template file to create its specific Kubernetes deployment. 
```

Comment:
(Optional) Kubernetes **yaml** template file for deploying ProActive node pods. This template file **informs** Kubernetes **about** the environment **needed to run** ProActive node. Each node **uses** this template file to create its specific Kubernetes deployment **environment**.